src/main/java/org/jitsi/videobridge/BitrateController.java

```diff
@@ -37,15 +38,34 @@
  * @author George Politis
  */
 public class BitrateController
+    extends PeriodicRunnable
```

Comment:
what's the difference between PeriodicRunnable and RecurringRunnable?  Does PeriodRunnable already implement RecurringRunnable?  Why not use that for the `Injection` class in https://github.com/jitsi/jitsi-videobridge/pull/418 ?